src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+            ListFooterComponent={
+              <>
+                {dappsMinimalDisclaimerEnabled ? (
+                  <Text style={styles.disclaimer}>{t('dappsDisclaimerAllDapps')}</Text>
+                ) : null}
+              </>
+            }
+            ListHeaderComponent={
+              <>
+                <DescriptionView
+                  title={t('dappsScreen.title')}
+                  message={t('dappsScreen.message')}
+                />
+                {/* Dapps Filtering*/}
+                <View style={{ paddingTop: Spacing.Thick24 }}>
+                  <ScrollView
+                    horizontal={true}
+                    // Expand the scrollview to the edges of the screen
+                    style={{ marginHorizontal: -Spacing.Thick24 }}
+                    showsHorizontalScrollIndicator={false}
+                    bounces={false}
+                    ref={horizontalScrollView}
+                  >
+                    {/* All Dapps Filter */}
+                    <DappFilterChip
+                      chipFilter={{ id: 'all', name: t('dappsScreen.allDapps') }}
+                      selectedFilter={selectedFilter}
+                      setFilter={setSelectedFilter}
+                      lastChip={false}
+                      key={'all'}
+                    />
+                    {/* Category Filter Chips */}
+                    {categories.map((category, idx) => {
+                      return (
+                        <DappFilterChip
+                          chipFilter={{ id: category.id, name: category.name }}
+                          selectedFilter={selectedFilter}
+                          setFilter={setSelectedFilter}
+                          lastChip={idx === categories.length - 1}
+                          key={category.id}
+                        />
+                      )
+                    })}
+                  </ScrollView>
+                </View>
+                {dappFavoritesEnabled && (
+                  <>
+                    <Text style={styles.sectionTitle}>{t('dappsScreen.favoriteDappsUpper')}</Text>
+                    <FavoriteDappsSection
+                      filter={selectedFilter}
+                      removeFilter={() => {
+                        setSelectedFilter({ id: 'all', name: t('dappsScreen.allDapps') })
+                        horizontalScrollView.current?.scrollTo({ x: 0, animated: true })
+                      }}
+                      onPressDapp={onSelectDapp}
+                    />
+                  </>
+                )}
+
+                {dappFavoritesEnabled && (
+                  <Text style={styles.sectionTitle}>{t('dappsScreen.allDappsUpper')}</Text>
+                )}
+              </>
+            }
+            style={styles.sectionList}
+            contentContainerStyle={{
+              padding: Spacing.Thick24,
+              paddingBottom: Math.max(insets.bottom, Spacing.Regular16),
+              flexGrow: 1,
```

Comment:
we have a stylesheet already, can we move any styles that don't require logic to the stylesheet? it's not just here, there's quite a few places in this component where we should remove inline styles